### DISCUSSION TITLE
LaTex in Markdown pages

### DISCUSSION THREAD

**Discussion by anquetos** [Avaiga/taipy#1353] [Q&A]
Is there a way to use LaTex within Makdown pages ?

**Comment by FlorianJacta** [comment]
No, you can't, unfortunately. LaTex is not part of Markdown. What is your use case?

**Comment by anquetos** [comment]
I have an app with a methodology page where some calculations are explained and I have an equation to display.

**Reply by FlorianJacta** [reply]
Great, this can be interesting! Would you want to create an issue with your needs and requirements for this? [You can click here to do so](https://github.com/Avaiga/taipy/issues/new?assignees=&labels=%E2%9C%A8New+feature&projects=&template=feature-request.yml&title=%3Cwrite+a+small+description+here%3E) or we can create the issue corresponding to our needs ourselves if you'd like

**Reply by anquetos** [reply]
I made an issue.

**Reply by FlorianJacta** [reply]
Fantastic! Thank you!
